indigo_app/static/javascript/indigo/views/document_editor.js

```diff
@@ -69,6 +70,10 @@
 
       this.setupRenderers();
       this.textEditor.getSession().setMode(this.parent.model.tradition().settings.grammar.aceMode);
+
+      // get the appropriate annotation style for the tradition
+      this.annotationGenerator = Indigo.annotations[this.parent.model.tradition().settings.annotationStyle];
+      this.annotationAmendingWork = this.getAnnotationAmendingWork(this.parent.model);
```

Comment:
I think we could safely call this `amendingWork`, since it's not annotation specific.

Reply:
I wanted to be careful because it's a guess at the amending work that brought about the current document (if there's more than one you're just getting the first) and can't be relied on by our future selves for anything other than annotations. but I guess I can make a comment to that effect?

Reply:
If it's ambiguous, then I suggest looking it up at the last moment when the menu item is selected. There's no advantage to storing it globally then. Additionally, it can change if the user changes the expression date for the current document (unlikely, but worth bearing in mind).